README.md

```diff
@@ -30,6 +30,10 @@ Once the variables have been set, run `pod install` to install the necessary dep
 The application contains the following important view controllers
 
 - **LoginViewController** - Handles user authentication based on a provided username / password.
+- **EmailViewController** - First step of user registration to use AWS S3 storage, enter a valid email address and confirm the same.
```

Comment:
I think the wording for these could be slightly different, maybe:

- EmailViewController: Handles user entry of the email address used to create a new account
- PasswordViewController: Handles user entry of the password used to create a new account
- SecurityQuestionViewController: Handles user selection of the security question used to create a new account
- CreateAccountViewController: Handles user entry of the security answer used to create a new account

?